dataprofiler/profilers/profiler_options.py

```diff
@@ -391,6 +398,10 @@ def _validate_helper(self, variable_path='NumericalOptions'):
             errors.append("{}: The numeric stats must toggle on sum,"
                           " variance, and skewness if kurtosis is "
                           "toggled on.".format(variable_path))
+        if histogram_disabled and not mad_disabled:
+            errors.append("{}: The numeric stats must toggle on histogram "
+                          "and quantiles if median absolute deviation is "
+                          "toggled on.".format(variable_path))
```

Comment:
Idk if I am just missing it but I think you should add this as a test case somewhere in the options test to ensure this gets caught.